charts/seed-bootstrap/charts/loki/values.yaml

```diff
@@ -45,12 +46,20 @@ readinessProbe:
 replicas: 1
 
 resources:
-  limits:
-    cpu: 450m
-    memory: 512Mi
-  requests:
-    cpu: 200m
-    memory: 300Mi
+  loki:
+    limits:
+      cpu: 450m
+      memory: 512Mi
+    requests:
+      cpu: 200m
+      memory: 300Mi
+  curator:
+    limits:
+      cpu: 200m
+      memory: 200Mi
+    requests:
+      cpu: 50m
+      memory: 100Mi 
```

Comment:
Can you reduce these values?
They are not scaled automatically and runs per cluster which may result in high compute costs for the seed clusters.